extensions/sql-bindings/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -5,294 +5,310 @@
 
 import * as vscode from 'vscode';
 import * as path from 'path';
-import { ConnectionDetails, IConnectionInfo } from 'vscode-mssql';
+import { ConnectionDetails, IConnectionInfo, IExtension } from 'vscode-mssql';
 import * as constants from '../common/constants';
 import * as utils from '../common/utils';
 import * as azureFunctionsUtils from '../common/azureFunctionsUtils';
 import { TelemetryActions, TelemetryReporter, TelemetryViews } from '../common/telemetry';
 import { BindingType } from 'sql-bindings';
 
-export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined): Promise<void> {
-	TelemetryReporter.sendActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.startAddSqlBinding);
+export class SqlBindingConfig {
```

Comment:
I created a class, because I thought we would use the mssql api in different functions, but it seems that it's only used when getting the connection string setting name